### DISCUSSION TITLE
How do you bump your Quarkus versions?

### DISCUSSION THREAD

**Discussion by praiaJaeyoungChung** [quarkusio/quarkus#31665] [Community]
Hi, we've started our project using Quarkus version 2.11.3 last year and we are seeing the latest version being at 2.16.x.

Sadly, contrary to my hopes, just bumping Quarkus version seems to break the builds. 

What are people doing to manage their Quarkus versions migrate upwards? Any tips and tricks you can share? 

Thanks

**Comment by maxandersen** [comment]
What is breaking for you?

First tip is migration guides https://github.com/quarkusio/quarkus/wiki/Migration-Guides

Second is that in quarkus 3 we are working on tooling to make upgrades easy.

**Reply by praiaJaeyoungChung** [reply]
My bad, I figured out my mistake; I didn't keep the build plugin version together with the platform version. All works well now between going from 2.11 --> 2.16,

**Reply by maxandersen** [reply]
In Curious on what exception You Got? Maybe something we could detect and give better error on?

**Reply by praiaJaeyoungChung** [reply]
I think I had plugin bumped to 2.16 and left platform at 2.11. 
The errors were coming from not having the directory structures as expected (integration tests, native tests etc)
